frontend/src/pages/v2/StaticCanvas.tsx

```diff
@@ -39,14 +39,21 @@ const StaticCanvas = ({ elements, layers, onLayersUpdate }: StaticCanvasProps) =
     reactFlowInstance.fitView();
   };
 
-  const doubleClickNode = (node: Node) => {
-    if (node.data['taskType'] !== TaskType.DAG) {
-      return;
-    }
-    const newLayers = [...layers, getTaskKeyFromNodeKey(node.id)]; //remove `task.`
+  const subDagExpand = (nodeKey: string) => {
+    const newLayers = [...layers, getTaskKeyFromNodeKey(nodeKey)]; //remove `task.`
```

Comment:
nit: what does the comment mean?

Reply:
This is a memo to say that the node key starts with `task.`, and `getTaskKeyFromNodeKey ()` is to remove this prefix from the node key. I will delete this comment to avoid confusion.